integrationTests/tesGameHelperFunctions.go

```diff
@@ -0,0 +1,259 @@
+package integrationTests
+
+import (
+	"encoding/hex"
+	"fmt"
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/ElrondNetwork/elrond-go/data/state"
+	"github.com/ElrondNetwork/elrond-go/data/transaction"
+	"github.com/stretchr/testify/assert"
+)
+
+// DeployScTx creates and sends a SC tx
+func DeployScTx(nodes []*TestProcessorNode, senderIdx int, scCode string) {
+	fmt.Println("Deploying SC...")
+	txDeploy := createTxDeploy(nodes[senderIdx].OwnAccount, scCode)
+	nodes[senderIdx].SendTransaction(txDeploy)
+	fmt.Println("Delaying for disseminating the deploy tx...")
+	time.Sleep(stepDelay)
+
+	fmt.Println(MakeDisplayTable(nodes))
+}
+
+func createTxDeploy(twa *TestWalletAccount, scCode string) *transaction.Transaction {
+	tx := &transaction.Transaction{
+		Nonce:    0,
+		Value:    big.NewInt(0),
+		RcvAddr:  make([]byte, 32),
+		SndAddr:  twa.PkTxSignBytes,
+		Data:     scCode,
+		GasPrice: 0,
+		GasLimit: 1000000000000,
+	}
+	txBuff, _ := TestMarshalizer.Marshal(tx)
+	tx.Signature, _ = twa.SingleSigner.Sign(twa.SkTxSign, txBuff)
+
+	return tx
+}
+
+func createTxEndGame(twa *TestWalletAccount, round int, scAddress []byte) *transaction.Transaction {
+	tx := &transaction.Transaction{
+		Nonce:    0,
+		RcvAddr:  scAddress,
+		SndAddr:  twa.PkTxSignBytes,
+		Data:     fmt.Sprintf("endGame@%d", round),
+		GasPrice: 0,
+		GasLimit: 1000000000000,
+	}
+	txBuff, _ := TestMarshalizer.Marshal(tx)
+	tx.Signature, _ = twa.SingleSigner.Sign(twa.SkTxSign, txBuff)
+
+	fmt.Printf("End %s\n", hex.EncodeToString(twa.PkTxSignBytes))
+
+	return tx
+}
+
+func createTxJoinGame(twa *TestWalletAccount, joinGameVal *big.Int, round int, scAddress []byte) *transaction.Transaction {
+	tx := &transaction.Transaction{
+		Nonce:    0,
+		Value:    joinGameVal,
+		RcvAddr:  scAddress,
+		SndAddr:  twa.Address.Bytes(),
+		Data:     fmt.Sprintf("joinGame@%d", round),
+		GasPrice: 0,
+		GasLimit: 1000000000000,
+	}
+	txBuff, _ := TestMarshalizer.Marshal(tx)
+	tx.Signature, _ = twa.SingleSigner.Sign(twa.SkTxSign, txBuff)
+
+	fmt.Printf("Join %s\n", hex.EncodeToString(twa.Address.Bytes()))
+
+	return tx
+}
+
+// PlayerJoinsGame creates and sends a join game transaction to the SC
+func PlayerJoinsGame(
+	nodes []*TestProcessorNode,
+	player *TestWalletAccount,
+	joinGameVal *big.Int,
+	round int,
+	scAddress []byte,
+) {
+	txDispatcherNode := getNodeWithinSameShardAsPlayer(nodes, player.Address.Bytes())
+	fmt.Println("Calling SC.joinGame...")
+	txScCall := createTxJoinGame(player, joinGameVal, round, scAddress)
+	txDispatcherNode.SendTransaction(txScCall)
```

Comment:
a time.Sleep(stepDelay) is required after this line. Might add also fmt.Println(MakeDisplayTable(nodes))

Reply:
fmt.Prinln after syncBlock.